examples/example1/example1.pack

```diff
@@ -0,0 +1,29 @@
+Content-Type: application/package
+Content-Location: http://example.org/example1.pack
+Link: </index.html>; rel=describedby
+
+--j38n02qryf9n0eqny8cq0
+Content-Location: /index.html
+Content-Type: text/html
+
+<body>
+  <a href="otherPage.html">Other page</a>
+</body>
+--j38n02qryf9n0eqny8cq0
+Content-Location: /otherPage.html
+Content-Type: text/html
+
+<body>
+  Hello World! <img src="images/world.png">
+</body>
+--j38n02qryf9n0eqny8cq0
+Content-Location: /images/bank.png
+Content-Type: image/png
+Content-Transfer-Encoding: base64
```

Comment:
Would be nice to avoid base64 from the start... would guthub barf if it was just a binary image?